### PR TITLE
[Testing] Enabling ported UITests from Xamarin.UITests to Appium #1

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Recently, We have completed porting the legacy UITests from Xamarin.Forms that still used Xamarin.UITest to .NET MAUI and Appium https://github.com/dotnet/maui/pull/24153, although we had a group that was commented or ignored. The causes are:
- A change is required in the example or test to adapt to some small change in .NET MAUI.
- An auxiliary extension method needs to be implemented in Appium to perform some action.
- The test fails due to a product failure.

The idea is to enable them all, implementing the actions that are necessary and also opening issues and making fixes where necessary.

We are going to enable tests in blocks in different PRs. This is the first group of tests enabled.

### CODE REVIEW

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25662.cs [hunk 1]
```diff
@@ -12,14 +13,23 @@ public Bugzilla25662(TestDevice testDevice) : base(testDevice)
...
+	public void Bugzilla25662Test()
+	{
+		if (App is not AppiumApp app2 || app2 is null || app2.Driver is null)
+		{
+			throw new InvalidOperationException("Cannot run test. Missing driver to run quick tap actions.");
+		}
+
+		App.WaitForNoElement("One");
+
+		var item = app2.Driver.FindElement(OpenQA.Selenium.By.XPath("//*[@text='" + "One" + "']"));
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25662.cs:30]: We can simplify this after merging https://github.com/dotnet/maui/pull/25177
This PR is in Draft status until be able to refactor this cases.

File: src/Controls/tests/TestCases.HostApp/Issues/Bugzilla/Bugzilla26233.cs [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
-using Microsoft.Maui.Controls.Internals;
-
 namespace Maui.Controls.Sample.Issues;
 
 [Issue(IssueTracker.Bugzilla, 26233, "Windows phone crashing when going back to page containing listview with Frame inside ViewCell")]
-public class Bugzilla26233 : TestContentPage
+public class Bugzilla26233 : NavigationPage
 {
-	protected override void Init()
+	public Bugzilla26233() : base(new MainPage())
```
Comment (PureWeen) [inline src/Controls/tests/TestCases.HostApp/Issues/Bugzilla/Bugzilla26233.cs:6]: I think we can remove base(new MainPage())

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25662.cs (hunk 1 above)
Comment (jfversluis) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla25662.cs]: That PR is merged! Should we update this first?

**Review by jfversluis — APPROVED** [review]

**Pull request merged**